lineman/app/components/user_avatar/user_avatar.haml

```diff
@@ -1,7 +1,5 @@
-.user-avatar{aria-hidden: 'true', class: '{{size}}'}
-  .avatar.avatar-initials{ng-if: 'user.avatarKind == "initials"', ng-class: '{ coordinator: coordinator }'}
+.user-avatar{aria-hidden: 'true', ng-class: '{"user-avatar--large": large, "user-avatar--small": small, "user-avatar--coordinator": coordinator}'}
```

Comment:
ng-class is way overkill here imho, especially given that we're going to want to add more sizes and that the directive doesn't need to be dynamic based on size.

```
$scope.size = 'small' unless $scope.size?
```
```
.user-avatar{class: 'user-avatar--{{size}}'}
```